src/App.js

```diff
@@ -1,20 +1,26 @@
 import './App.css';
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
 import Home from './components/Home';
-import Books from './components/Books';
 import Components from './components/Components';
-
+import Books from './components/Books';
+import storeConfig from './redux/store';
 // end imports
 
+const store = storeConfig();
+
 function App() {
   return (
-    <Router>
-      <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="/books" element={<Books />} />
-        <Route path="/categories" element={<Components />} />
-      </Routes>
-    </Router>
+    <Provider store={store}>
+      <Router>
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/books" element={<Books />} />
```

Comment:
- [ ] [OPTIONAL] I have noticed that your root path does not start from the book page. I would recommend changing your root path to direct to the book component as originally your app should start from this page.